<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBoxcameraconnect">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxpreview">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxcapture">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxautofocus">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxiso">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelcameraconnect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>42</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>相机连接</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelpreview">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>92</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>实时预览</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelcapture">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>152</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>图像采集</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelautofocus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>212</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>自动聚焦</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeliso">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>282</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ISO</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
